MyHeartCounts/Onboarding/FinalEnrollmentStep.swift

```diff
@@ -30,34 +32,63 @@ struct FinalEnrollmentStep: View {
     
     @State private var viewState: ViewState = .idle
     
+    
+    private var showTrialSection: Bool {
+        onboardingData.consentResponses?.selects["short-term-physical-activity-trial"] == "short-term-physical-activity-trial-yes"
+    }
+    
     var body: some View {
-        OnboardingView {
-            OnboardingTitleView(title: "Welcome to My Heart Counts")
-        } content: {
-            let doc = MarkdownDocument(
-                metadata: [:],
-                blocks: [.markdown(id: nil, rawContents: loadText())]
-            )
-            MarkdownView(markdownDocument: doc)
-        } footer: {
+        VStack {
+            ScrollView {
+                VStack(alignment: .leading, spacing: 8) {
```

Comment:
@lukaskollmer This combination of `VStack`s, `ScrollView`s and paddings/spacing is reused a lot; we might want to generalize this as part of Spezi Onboarding at some point.